### PR TITLE
A record is modified to intended CNAME

### PR DESCRIPTION
**Pull request by davestys**
A record converted to intended CNAME record.

### CODE REVIEW

File: aweber.com.landing-pages-subdomain.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
    "variableDescription":"",
    "records":[
        {
-         "type": "A",
+         "type": "CNAME",
```
Comment (pawel-kow) [inline aweber.com.landing-pages-subdomain.json:12]: CNAME on APEX is not allowed. If intended usage is only for subdomains, please set `hostRequired` to `true`.

Comment (davestys) [reply aweber.com.landing-pages-subdomain.json:12]: ah that's it. thanks

Comment (davestys) [reply aweber.com.landing-pages-subdomain.json:12]: So, best if I use a variable like %subdomain%:
`"host": "%subdomain%"`

**Comment by pawel-kow** [comment]
@davestys, actually I mean you should add `hostRequired` parameter and leave the host as it was with `@`

**Comment by davestys** [comment]
ok will do.

**Comment by davestys** [comment]
@pawel-kow is there anything else I need to do to have this update merged? Thanks

**Review by pawel-kow — APPROVED** [review]

**Comment by pawel-kow** [comment]
Done

**Comment by davestys** [comment]
thanks!

**Pull request merged**